app/services/activities/import_from_csv.rb

```diff
@@ -288,6 +288,7 @@ def convert_to_attributes
         attributes[:recipient_region] ||= inferred_region
         attributes[:call_present] = (@row["Call open date"] && @row["Call close date"]).present?
         attributes[:sector_category] = get_sector_category(attributes[:sector])
+        attributes[:status] = infer_status(attributes)
```

Comment:
Just simple curiosity... Is there a reason why we pass all attributes here as an argument instead of passing just `programme_status`?

Reply:
No reason, other than consistency with the other `infer_` method!